docs/user/localizing-your-projects.rst

```diff
@@ -69,9 +69,14 @@ following required fields:
 5. **Download prefix**: used for manually downloading files. Select any
    localizable file on GitHub, click ``Raw`` and replace locale code and the
    following bits in the URL with {locale_code}.
-6. Click **SAVE PROJECT** at the bottom of the page.
-7. After the page reloads, click **SYNC** and wait for Pontoon to import
-   strings. You can monitor the progrss in the Sync log (``/sync/log/``).
+7. Click **SAVE PROJECT** at the bottom of the page.
```

Comment:
That's actually 6, not 7?

Reply:
It's doesn't matter much, as when processed, those numbers are not used (they are only cosmetic, for when you read the source file). You could have all items start with `1.`, the end result will be a correctly numbered list.

Reply:
Yeah, converting all to 1. is even better.